airbyte-integrations/csv-destination/src/test-integration/java/io/airbyte/integrations/destination/csv/CsvDestinationIntegrationTest.java

```diff
@@ -60,7 +61,7 @@ protected JsonNode getInvalidConfig() {
   }
 
   @Override
-  protected List<JsonNode> recordRetriever(TestDestinationEnv testEnv) throws Exception {
+  protected Set<JsonNode> recordRetriever(TestDestinationEnv testEnv, String streamName) throws Exception {
```

Comment:
You should keep List and sort or create an assert that is not sensitive to order. Set will make it hard to test for duplicated values.

Reply:
good point.